base/shell/explorer/settings.cpp

```diff
@@ -82,7 +82,6 @@ BOOL TaskbarSettings::Load()
         sr.AlwaysOnTop = TRUE;
         sr.SmallIcons = TRUE;
```

Comment:
"SmSmallIcons", and set it to FALSE by default (i.e. default is large icons)

Reply:
```suggestion
        sr.SmallIcons = FALSE;
```